### ISSUE TITLE
turn TCP_FASTOPEN back on if libcurl is new enough?

### ISSUE THREAD

**Issue by StefanKarpinski** [JuliaLang/Downloads.jl#83]
See https://github.com/JuliaLang/Downloads.jl/pull/76. For libcurl ≥ 7.74 it should be safe to turn on TCP_FASTOPEN even for FTP requests.

**Comment by StefanKarpinski** [comment]
I do want to do this, but we're not using a recent enough libcurl for this to be fixed, so I'm still waiting on that so that a) will actually benefit from this and b) can test both code paths with different Julia versions.

**Comment by tecosaur** [comment]
Now that 7.74 has been out for a while, would it be worth setting TCP_FASTOPEN again?

**Comment by StefanKarpinski** [comment]
Yeah, seems like we could, although it's a little unclear to me if it makes that much difference for downloads.
